.github/workflows/release.yml

```diff
@@ -48,3 +48,12 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+
+      - name: Get Package Version
+        run: echo "WRANGLER_VERSION=$(npm view wrangler version)" >> $GITHUB_ENV
+        working-directory: packages/wrangler
+
+      - name: SourceMapping Rewrite & Sentry Upload
+        run: node .github/sentry-upload.js
+      - run: npx sentry-cli --auth-token ${{ secrets.SENTRY_AUTH_TOKEN }} releases --org jacob-mg-evans --project nodejs files "wrangler@$WRANGLER_VERSION" upload-sourcemaps ./wrangler-dist --url-prefix /
+        working-directory: packages/wrangler
```

Comment:
Tested these scripts locally on a fork where I was running the Actions 
<img width="1374" alt="Screen Shot 2022-01-24 at 2 52 34 PM" src="https://user-images.githubusercontent.com/27247160/150863210-e9fc23ab-999c-402c-9b29-058dfd9b871d.png">